packages/zwave-js/src/lib/serialapi/capability/GetSerialApiInitDataMessages.ts

```diff
@@ -215,3 +218,115 @@ export class GetSerialApiInitDataResponse extends Message {
 //                         is SUC:             true
 //                         chip type:          7
 //                         chip version:       0
+
+export interface GetLongRangeNodesRequestOptions extends MessageBaseOptions {
+	listStartOffset128: number;
+}
+
+@messageTypes(MessageType.Request, FunctionType.GetLongRangeNodes)
+@expectedResponse(FunctionType.GetLongRangeNodes)
+@priority(MessagePriority.Controller)
+export class GetLongRangeNodesRequest extends Message {
+	public constructor(
+		host: ZWaveHost,
+		options:
+			| MessageDeserializationOptions
+			| GetLongRangeNodesRequestOptions,
+	) {
+		super(host, options);
+
+		if (gotDeserializationOptions(options)) {
+			// BUGBUG: validate length at least
```

Comment:
Some existing code validates buffer lengths, some does not. I'm assuming I should be calling `ValidateBuffer` or whatever here...?

Reply:
We generally validate buffer lengths for the command classes, as there are tons of nodes that send broken frames. The controller firmware is generally much more predictable.